source/code/source/playfab/QoS/XPlatSocket.cpp

```diff
@@ -107,7 +107,7 @@ namespace PlayFab
                 return GetLastErrorCode();
             }
 
-            auto inAddr = (struct in_addr*) he->h_addr_list[0];
+            in_addr* inAddr = (struct in_addr*) he->h_addr_list[0];
```

Comment:
> (struct in_addr*) [](start = 29, length = 18)

nit: you're using c++ so I recommend using a c++ style cast

`reinterpret_cast<in_addr*>(he->h_addr_list[0]);`